pkg/cli/adbtunnel.go

```diff
@@ -160,19 +165,36 @@ type ADBForwarder struct {
 	listener  net.Listener
 	conn      net.Conn
 	port      int
-	status    string
+	status    int
 	statusMtx sync.Mutex
 	logger    *log.Logger
 }
 
 const (
-	ADBFwdInitializing = "initializing"
-	ADBFwdReady        = "ready"
-	ADBFwdConnected    = "connected"
-	ADBFwdStopped      = "stopped"
-	ADBFwdFailed       = "failed"
+	ADBFwdInitializing = 0 // The initial state after creation
```

Comment:
this seems a good use case to use https://go.dev/ref/spec#Iota

Reply:
Done